### ISSUE TITLE
crash when deleting peg

### ISSUE THREAD

**Issue by phillycheese007** [opentoonz/opentoonz#2274]
### Issue Summary

i try to delete a peg in a scene and it crashes every time tried this like 5 times in a row same thing every time
### Screenshot or Video Reference

Im gonna be a little lazy and not record a new video of this...(i accidently cut the end of the video...after the i click you you see the spinning wheel of death and then it crashes)
![fraz5](https://user-images.githubusercontent.com/25863731/45059334-a456b280-b069-11e8-9c67-d87e75b7f74a.gif)

### Steps to Reproduce

im not exactly sure as simply deleting a peg does not usually cause this issue. Please let me know if you would like the scene file
### System Information
* **OpenToonz Version**: latest nightly
* **Operating System**: windows 10
* **CPU**: 
* **Memory**: 
* **Graphics Card**: 
* **Graphics Tablet**:

**Comment by manongjohn** [comment]
@phillycheese007 Can you provide the scene so we can take a look at it?

**Comment by RodneyBaker** [comment]
Yes, please.  Share the scene file!  I wish more people would do that.  :)

**Comment by phillycheese007** [comment]
this might be a while....gonna be some tracking down......dealing with
housing issues at the moment
